### ISSUE TITLE
Ability for the magic items to carry over the item macro and automation features.

### ISSUE THREAD

**Issue by ChobitsCrazy** [PwQt/magic-items-2#81]
Items With Spells is able to pull over the item macro and MIDI properties, would like to see the same functionality within magic items 2.

**Comment by MotoMoto1234** [comment]
Mine carry over just fine in the midi beta and newer MIM2 versions.

**Comment by ChobitsCrazy** [comment]
Hmm maybe because I'm on the older versions. Haven't moved to 3.X yet. Let me try it out on my test env.

**Comment by ChobitsCrazy** [comment]
Yeah ran a test case in the latest version and it doesn't appear to be working for me.
Example: Ensnaring Strike on a Ring
Foundry: v11.315
DND5e: v3.0.4
Magic Items 2: v1.6.3.1

Spell is set up with ItemMacro and set to execute with ItemMacro
![image](https://github.com/PwQt/magic-items-2/assets/21301107/493ae774-d43a-42ac-a6b1-0f8bc52f1b78)
![image](https://github.com/PwQt/magic-items-2/assets/21301107/4b600b07-acaf-4350-b557-a96b43ce1d30)

Putting the spell directly on the actor causes the effect to be added and execute correctly.
![image](https://github.com/PwQt/magic-items-2/assets/21301107/3515569d-197b-4717-85bb-4eba66bb6808)
![image](https://github.com/PwQt/magic-items-2/assets/21301107/e21e1eb5-e57d-4c2e-8a8a-b5782e0e13d2)

However when used within an item (Ring) the effects do not take place.
![image](https://github.com/PwQt/magic-items-2/assets/21301107/44e9e7c6-832d-4ba3-b2bf-3fb276890904)
![image](https://github.com/PwQt/magic-items-2/assets/21301107/17adf8b1-c13a-4007-bdc3-0c62e29f05b5)
![image](https://github.com/PwQt/magic-items-2/assets/21301107/8e501cc1-8258-4bc8-bd8c-7f2da0c96de1)
![image](https://github.com/PwQt/magic-items-2/assets/21301107/39c0e3d8-fc9f-43aa-a23d-c55696ee9a1a)
